server/store/ip_store.go

```diff
@@ -75,10 +75,10 @@ func RegisterIPStoreDriver(name string, driver IPStoreDriver) {
 }
 
 // OpenIPStore returns an IPStore specified by a configuration.
-func OpenIPStore(cfg *DriverConfig) (IPStore, error) {
+func OpenIPStore(cfg *DriverConfig) (IPStore, StopperFunc, error) {
```

Comment:
I'm not feeling good about this - store just embeds IPStore, StringStore and PeerStore, all of them would have a `Stop()` func, plus the store itself has one. I'm not familiar with how go handles that